src/renderer/components/SettingsModal.tsx

```diff
@@ -297,11 +298,54 @@ const EnvironmentSettings = memo(() => {
         }
     }, [startupTemplate, lastDirectory, setStartupTemplate]);
 
+    const [reservedSystemMemory, setReservedSystemMemory] = useReservedSystemMemory;
+
+    // Maximum amount reserved for the system is 80 % of the total memory
+    const calculateMaxValue = () => (totalMemory / 1024 ** 2) * 0.8;
+    const minValue = 8192;
+
     return (
         <VStack
             divider={<StackDivider />}
             w="full"
         >
+            {isArmMac ? (
+                <SettingsItem
+                    description={`The amount of RAM reserved for the system, that will not be used by chaiNNer for upscaling. (${
+                        minValue / 1024
```

Comment:
@RunDevelopment 

I should calculate the min value here so that it can never be more than 20% of the system's total memory.